### PR TITLE
Package structure, tests and workflows.

### PR DESCRIPTION
**Pull request by tetov**

### CODE REVIEW

**Comment by tetov** [comment]
Tests on GH Actions pass.

https://github.com/tetov/unnamed-irc-bridge/actions/runs/751904035

**Comment by tetov** [comment]
A bare bones package setup using setuptools and (nearly) [setup.cfg-only project](https://setuptools.readthedocs.io/en/latest/setuptools.html#id4). While more elegant solutions exists such as flit and poetry, this setup doesn't require any additional packages except the standard setuptools. 

I think a neater import structure should be implemented, but that will have to be in the next PR.

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,28 @@
+[metadata]
+name = unnamed_irc_bridge
+description = "Matrix IRC bridge"
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/hifi/unnamed_irc_bridge
```
Comment (hifi) [inline setup.cfg:6]: This URL is wrong.

**Review by hifi — CHANGES_REQUESTED** [review]
Other than the wrong URL all this seems good. I'd like to name this project first, though so the temp name doesn't get embedded in so many places.

So if you don't mind let's figure out a name for this and merge after that?

**Pull request merged**